Exporter/src/test/java/uk/ac/ebi/biostd/exporter/StatsExportTest.java

```diff
@@ -56,17 +65,22 @@ public void setup() throws IOException {
 
         new File(basePath + "/updates").mkdir();
         when(properties.getOutFilePath()).thenReturn(basePath + "/updates/stats.csv");
-
-        String pageTabFile = basePath + "/submissions/S-EPMC/S-EPMCxxx986/S-EPMC4273986/S-EPMC4273986.pagetab.tsv";
-        FileUtils.copyFile(getResource("/test_files/S-EPMC4273986.tsv"), new File(pageTabFile));
     }
 
     @Test
     public void testStatsExport() throws Exception {
         exportPipeline.execute();
         File statsFile = new File(folder.getRoot().getAbsolutePath() + "/updates/stats.csv");
         assertThat(statsFile).exists();
+        assertStats(statsFile, EXPECTED_STATS_1, EXPECTED_STATS_2);
     }
 
-}
+    private void assertStats(File statsFile, String... expectedStats) throws Exception {
```

Comment:
Tiny detail, but this method is call assertStats and actually recieve a file, the correct think to do IMHO will be pass the list of lines to this method.

Reply:
Fixed